__tests__/app.test.js

```diff
@@ -79,3 +80,30 @@ describe("GET/articles/:article_id", () => {
       });
   });
 });
+
+describe("Get /api/articles", () => {
+  test("Status 200 - responds with an array of all the article objects sorted by date in descending order. ", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles.length).toBe(12);
+        expect(articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+
+        body.articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("number");
+          expect(articles.body).toBe(undefined);
+        });
```

Comment:
Alternatively to this approach a couple matchers you can look into are .objectContaining and .toMatchObject. These also allow for later extension of the endpoint as well making then very flexible matchers.